app/packages/operators/src/state.ts

```diff
@@ -339,7 +345,7 @@ export const useOperatorPrompt = () => {
       operator.isRemote ? RESOLVE_TYPE_TTL : 0,
       { leading: true }
     ),
-    [cachedResolvedInput, setResolvedCtx, ctx]
+    [cachedResolvedInput, setResolvedCtx, ctx, operatorURI]
```

Comment:
Is memoizing operatorURI required? 🤔 
```suggestion
    [cachedResolvedInput, setResolvedCtx, ctx, operator.uri]
```

Reply:
No, it's probably not necessary. Will update it as part of rebase 👍